lib/music_library_controller.rb

```diff
@@ -22,84 +21,80 @@ def application_loop
   end
 
   def check_input(command)
-    if options.include? command
-      check_for_additional_input(command)
-      send(options[command])
-    else
+    send(command.tr(" ", "_"))
+    rescue NoMethodError
       cli.undefined
-    end
-  end
-
-  def check_for_additional_input(command)
```

Comment:
Instead of returning nothing for the commands that don't need a second input, just check those that need a second input and issue the command for them